sdk/src/process_instruction.rs

```diff
@@ -24,7 +24,6 @@ pub type LoaderEntrypoint = unsafe extern "C" fn(
     invoke_context: &dyn InvokeContext,
 ) -> Result<(), InstructionError>;
 
-pub type ProcessInstruction = fn(&Pubkey, &[KeyedAccount], &[u8]) -> Result<(), InstructionError>;
 pub type ProcessInstructionWithContext =
```

Comment:
Why not just call this `ProcessInstruction` now since there only needs to be one?

Reply:
will do!

Reply:
I think there's value in keeping ProcessInstructionWithContext.
There's `solana_program::entrypoint::ProcessInstruction`, which has no context.   So keeping this as 
`ProcessInstructionWithContext` improves readability especially in files that use both types